docs/source/getting_started/handling_time.ipynb

```diff
@@ -0,0 +1,634 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "240c7f4d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "window_fm, window_features = ft.dfs(entityset=es,\n",
+    "                                    target_dataframe_name=\"customers\",\n",
+    "                                    cutoff_time=cutoff_times,\n",
+    "                                    cutoff_time_in_index=True,\n",
+    "                                    training_window=\"2 hour\")\n",
+    "\n",
+    "window_fm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4efca117",
+   "metadata": {},
+   "source": [
+    "We can see that that the counts for the same feature are lower after we shorten the training window:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f044c9f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm[[\"COUNT(transactions)\"]]\n",
+    "\n",
+    "window_fm[[\"COUNT(transactions)\"]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d430509",
+   "metadata": {},
+   "source": [
+    "## Setting a Last Time Index\n",
+    "\n",
+    "The training window in Featuretools limits the amount of past data that can be used while calculating a particular feature vector. A row in the dataframe is filtered out if the value of its time index is either before or after the training window. This works for dataframes where a row occurs at a single point in time. However, a row can sometimes exist for a duration.\n",
+    "\n",
+    "For example, a customer's session has multiple transactions which can happen at different points in time. If we are trying to count the number of sessions a user has in a given time period, we often want to count all the sessions that had *any* transaction during the training window. To accomplish this, we need to not only know when a session starts, but also when it ends. The last time that an instance appears in the data as the ``_ft_last_time`` column that featuretools has calculated for every row in the dataframe. We can compare the time index and the last time index of the ``sessions`` dataframe above:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "87b8ce53",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "last_time_index_col = es['sessions'].ww.metadata.get('last_time_index')\n",
+    "es['sessions'][['session_start', last_time_index_col]].head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cdcf563",
+   "metadata": {},
+   "source": [
+    "Featuretools can automatically add last time indexes to every DataFrame in an ``Entityset`` by running ``EntitySet.add_last_time_indexes()``. If a ``last_time_index`` has been set, Featuretools will check to see if the ``last_time_index`` is after the start of the training window. That, combined with the cutoff time, allows DFS to discover which data is relevant for a given training window.\n",
```

Comment:
> If a ``last_time_index`` has been set, Featuretools will check to see if the ``last_time_index`` is after the start of the training window.

This PR didn't touch this sentence, but I wonder if adding a clarification at the beginning of the sentence would help orient the reader back to how LTIs are used from the specifics of how they are created

> When using a training window, if a ``last_time_index`` has been set, Featuretools will check to see if the ``last_time_index`` is after the start of the training window.

Reply:
I like the addition, adding!